scripts/windows/build-win-application.sh

```diff
@@ -120,10 +120,6 @@ touch "$BUILDBASE"/requirements.txt
 echo "
 #:wheel: scikit-learn https://pypi.python.org/packages/b8/9a/02d5d76be66c57aaa9f917c87007b9b0bf486992cc7701512464d1ce11e9/scikit_learn-0.17.1-cp34-cp34m-win32.whl#md5=ab00daed7cdac4cb16ad0613b91be07e
 scikit-learn==0.17.1
-
-#:wheel: Bottlecheset https://dl.dropboxusercontent.com/u/100248799/Bottlechest-0.7.1-cp34-none-win32.whl#md5=629ba2a148dfa784d0e6817497d42e97
---find-links https://dl.dropboxusercontent.com/u/100248799/Bottlechest-0.7.1-cp34-none-win32.whl
-Bottlechest==0.7.1
```

Comment:
>  Is it ok just removing bottlechest installation step like that? 

Yes, but also note that bottlechest is referenced in a few more places in this script

> Since Bottleneck upstream does not prebuild wheels, where do we build it?

For Bottlechest, I build the wheels in a local windows VM.
But the major reason that Bottlechest was prebuild was that it is so slow to compile, Bottleneck (at least the current 1.1 release) compiles much faster, so I guess it could be build in this script.